serialization/api.go

```diff
@@ -414,53 +414,53 @@ type PortableReader interface {
-	FieldKindArrayOfNullableInt16         FieldKind = 37
-	FieldKindNullableInt32                FieldKind = 38
-	FieldKindArrayOfNullableInt32         FieldKind = 39
-	FieldKindNullableInt64                FieldKind = 40
-	FieldKindArrayOfNullableInt64         FieldKind = 41
-	FieldKindNullableFloat32              FieldKind = 42
-	FieldKindArrayOfNullableFloat32       FieldKind = 43
-	FieldKindNullableFloat64              FieldKind = 44
-	FieldKindArrayOfNullableFloat64       FieldKind = 45
-	FieldKindNotAvailable                 FieldKind = 46
+	FieldKindNotAvailable   FieldKind = 0
+	FieldKindBoolean        FieldKind = 1
+	FieldKindArrayOfBoolean FieldKind = 2
+	FieldKindInt8           FieldKind = 3
+	FieldKindArrayOfInt8    FieldKind = 4
+	// FieldKindChar                      FieldKind = 5
+	// FieldKindArrayOfChar               FieldKind = 6
+	FieldKindInt16                        FieldKind = 7
+	FieldKindArrayOfInt16                 FieldKind = 8
+	FieldKindInt32                        FieldKind = 9
+	FieldKindArrayOfInt32                 FieldKind = 10
+	FieldKindInt64                        FieldKind = 11
+	FieldKindArrayOfInt64                 FieldKind = 12
+	FieldKindFloat32                      FieldKind = 13
+	FieldKindArrayOfFloat32               FieldKind = 14
+	FieldKindFloat64                      FieldKind = 15
+	FieldKindArrayOfFloat64               FieldKind = 16
+	FieldKindString                       FieldKind = 17
+	FieldKindArrayOfString                FieldKind = 18
+	FieldKindDecimal                      FieldKind = 19
+	FieldKindArrayOfDecimal               FieldKind = 20
+	FieldKindTime                         FieldKind = 21
+	FieldKindArrayOfTime                  FieldKind = 22
+	FieldKindDate                         FieldKind = 23
+	FieldKindArrayOfDate                  FieldKind = 24
+	FieldKindTimestamp                    FieldKind = 25
+	FieldKindArrayOfTimestamp             FieldKind = 26
+	FieldKindTimestampWithTimezone        FieldKind = 27
+	FieldKindArrayOfTimestampWithTimezone FieldKind = 28
+	FieldKindCompact                      FieldKind = 29
+	FieldKindArrayOfCompact               FieldKind = 30
+	// FieldKindPortable                  FieldKind = 31
+	// FieldKindArrayOfPortable           FieldKind = 32
+	FieldKindNullableBoolean        FieldKind = 33
+	FieldKindArrayOfNullableBoolean FieldKind = 34
+	FieldKindNullableInt8           FieldKind = 35
+	FieldKindArrayOfNullableInt8    FieldKind = 36
+	FieldKindNullableInt16          FieldKind = 37
+	FieldKindArrayOfNullableInt16   FieldKind = 38
+	FieldKindNullableInt32          FieldKind = 39
+	FieldKindArrayOfNullableInt32   FieldKind = 40
+	FieldKindNullableInt64          FieldKind = 41
+	FieldKindArrayOfNullableInt64   FieldKind = 42
+	FieldKindNullableFloat32        FieldKind = 43
+	FieldKindArrayOfNullableFloat32 FieldKind = 44
+	FieldKindNullableFloat64        FieldKind = 45
+	FieldKindArrayOfNullableFloat64 FieldKind = 46
```

Comment:
why these look like 
```
-
   -
 -
```
?

Can you fix the formatting?

Reply:
That's automatically formatted by gofmt, we have no control over it.